src/skewsymmetricmv.jl

```diff
@@ -26,48 +26,60 @@ using VectorizationBase, LinearAlgebra, Static
       Nr = N ÷ $CU
       @assert Nr * $CU == N# "TODO: handle remainders!!!"
       r = 0
-      @show Nr
-      for ro = 1:Nr
+      for ro = 1:1
```

Comment:
??

Reply:
It was to limit the shown elements trying to debug